### ISSUE TITLE
As a user I want to be able to use the Liquid Formatter on *.liquid files only.

### ISSUE THREAD

**Issue by symonxdd** [panoply/vscode-liquid#81]
I can't believe this is not an option in the extension settings... I've got both html and liquid files which require their own formatter.
What a shame.

Is it possible to add this feature? Could I be of help, I'm myself an IT student and have some knowledge on web languages, though not yet in TS.

**Comment by panoply** [comment]
You can do this in your workspace settings:

```
{
 "[liquid]": {
    "editor.defaultFormatter": "sissel.shopify-liquid"
  }
}
```

> Is it possible to add this feature? Could I be of help, I'm myself an IT student and have some knowledge on web languages, though not yet in TS.

Thanks for your interest. I am looking for help but you might not be the right fit just yet.

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)
